test/integration/tests/1265-extensible-snapshots/Main.hs

```diff
@@ -1,8 +1,10 @@
-import StackTest
+import           StackTest
 
 main :: IO ()
 main = do
     stack ["build", "async"]
     stackErr ["build", "zlib-bindings"]
     stack ["build", "--stack-yaml", "stack-modify-lts.yaml", "async"]
+    stack ["build", "--stack-yaml", "stack-local-snapshot.yaml", "async"]
```

Comment:
I think maybe to test this case it would need to try to build `package`, right?  Also to test that the paths are correctly local, it would need to use `withCurrentDirectory`

Reply:
I think building package is not necessary (actually not possible, `package` merely contains a valid `.cabal` file).

But you are right, the file system layout does probably not require the fix... I'll add a commit to push snapshots and package one level deeper.